dotcom-rendering/src/types/content.ts

```diff
@@ -736,8 +736,8 @@ interface CampaignField {
 	description?: string;
 	required: boolean;
 	textSize?: number;
-	hideLabel: boolean;
-	hidden: boolean;
+	hideLabel?: boolean;
+	hidden?: boolean;
```

Comment:
by making these fields optional, we are able to make use of the exisiting CampaignFieldTextArea interface for message us, reducing the need to add more types.

Reply:
Nice 👍